src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodoId, setSelectedTodoId] = useState<number>(0);
+  const [filter, setFilter] = useState(Status.all);
+  const [query, setQuery] = useState('');
+  const getTodoByID = (id: number): Todo => {
+    return todos.find(todo => todo.id === id)!;
```

Comment:
```suggestion
    return todos.find(todo => todo.id === id);
```
if todos will be not find - function return undefined
and it will fix this linter issue
![Screenshot from 2023-08-19 16-02-45](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/33029864/f62d9aa7-66cd-47d8-8f0a-1d896b858fde)